src/System.Private.ServiceModel/tools/scripts/BuildCertUtil.cmd

```diff
@@ -1,41 +1,14 @@
-@echo off
-setlocal
-
-set _VSWHERE="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
-if exist %_VSWHERE% (
-  for /f "usebackq tokens=*" %%i in (`%_VSWHERE% -latest -prerelease -property installationPath`) do set _VSCOMNTOOLS=%%i\Common7\Tools
-)
-if not exist "%_VSCOMNTOOLS%" set _VSCOMNTOOLS=%VS140COMNTOOLS%
-if not exist "%_VSCOMNTOOLS%" goto :MissingVersion
-
-set VSCMD_START_DIR="%~dp0"
-call "%_VSCOMNTOOLS%\VsDevCmd.bat"
-goto :EnvSet
-
-:MissingVersion
-:: Can't find VS 2017, 2019
-echo Error: %~nx0 requires Visual Studio 2017 or 2019.
-echo        Please see https://github.com/dotnet/wcf/wiki/Developer-Guide for build instructions.
-exit /b 1
+@echo on
 
```

Comment:
I presume this was turned on for debugging purposes. Unless you have a specific reason to leave echo on, it should be turned off.

Reply:
I see. You're right it's turning on for debug only. Turning off.